src/connectors/Office365.ts

```diff
@@ -0,0 +1,131 @@
+import * as axios from "axios";
+import { all } from "bluebird";
+import * as fs from "fs-extra";
+import * as _ from "lodash";
+import * as path from "path";
+import * as qs from "qs";
+import { IFileContent } from "../Schema";
+import { buildFromLocalExcel } from "./Excel";
+
+declare interface IAzureSecret {
+  client_secret?: string;
+  client_id?: string;
+  username?: string;
+  password?: string;
+  tenant_directory?: string;
+}
+
+const OFFICE_PATH = ".voxa/office";
```

Comment:
probably better to use https://nodejs.org/api/fs.html#fs_fs_mkdtemp_prefix_options_callback

Reply:
I don't like the idea, because it'll be harder to track down.

Reply:
We want to use this folder for other purposes like storing expandable utterances, etc.

Reply:
Done!